cmd/res/configuration.yaml

```
@@ -0,0 +1,23 @@
+MaxEventSize: 0 # value 0 unlimit the maximum event size that can be sent to message bus or core-data
+
+Writable:
+  LogLevel: INFO
+
+# uncomment when running from command-line in hybrid mode with -cp -o flags
+Registry:
+  Host: localhost
+
+Clients:
+  core-metadata:
+    Host: localhost # uncomment when running from command-line in hybrid mode
```

Comment out so that it is included in above block
```suggestion
Clients:
  core-metadata:
    Host: localhost
```